csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs

```diff
@@ -67,6 +74,196 @@ public void CanCreateAndDisposeSessionWithModelPath()
             }
         }
 
+#if NET8_0_OR_GREATER
+#pragma warning disable SYSLIB5001 // System.Numerics.Tensors is only in preview so we can continue receiving API feedback
+        [Theory]
+        [InlineData(GraphOptimizationLevel.ORT_DISABLE_ALL, true)]
+        [InlineData(GraphOptimizationLevel.ORT_DISABLE_ALL, false)]
+        [InlineData(GraphOptimizationLevel.ORT_ENABLE_EXTENDED, true)]
+        [InlineData(GraphOptimizationLevel.ORT_ENABLE_EXTENDED, false)]
+        private void CanRunInferenceOnAModelDotnetTensors(GraphOptimizationLevel graphOptimizationLevel, bool enableParallelExecution)
+        {
+            var model = TestDataLoader.LoadModelFromEmbeddedResource("squeezenet.onnx");
+
+            using (var cleanUp = new DisposableListTest<IDisposable>())
+            {
+                // Set the graph optimization level for this session.
+                SessionOptions options = new SessionOptions();
+                cleanUp.Add(options);
+                options.GraphOptimizationLevel = graphOptimizationLevel;
+                if (enableParallelExecution) options.ExecutionMode = ExecutionMode.ORT_PARALLEL;
+
```

Comment:
We no longer optimize for it. It may disappear soon.

Reply:
What do you mean by this? 
I have seen some models run faster when parallel, and others run faster when serial. 
Faster == more inference/s in a server API scenario.

Reply:
It is possible. However, the kernels are currently implemented and optimized for intra parallelism. Meaning, each kernel computes how many threads it is going to use to speed up computations. In many cases it is going to conflict with parallel. I recommend SEQUENTIAL as a default (which it is), unless, of course, you have perf results that demonstrate otherwise. Certainly, PARALLEL is not a good example to put into test code.